portal-impl/src/com/liferay/portlet/documentlibrary/lar/FileEntryStagedModelDataHandler.java

```diff
@@ -537,7 +538,7 @@ protected void doImportStagedModel(
 					fileEntry.getTitle(), fileEntry.getDescription(), null, is,
 					fileEntry.getSize(), serviceContext);
 			}
-			catch (DuplicateFileException dfe) {
+			catch (DuplicateFileException | DuplicateFileEntryException dfe) {
```

Comment:
@adolfopa This DuplicateFileException can be removed, we only want to rename title when DuplicateFileEntryException is thrown inside addFileEntry, if a DuplicateFileException is received, a title rename of fileEntry will not help to fix the problem during **DLAppLocalServiceUtil.addFileEntry**